mautic/mautic
# UI/UX bug fixes

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR fixes bugs.

- X close button was behind the text in flash messages
- no padding for conditions and decisions on campaign builder
- missing confirmation for reset learning button in account settings
- removes hardcoded stripes for timeline table in contact details
- fixes modals permanently hidden after dismissed

![image](https://github.com/user-attachments/assets/1b4d45e5-8a71-44d2-a50a-d20d7ec1158b)

![image](https://github.com/user-attachments/assets/1fd5f015-75be-4035-b460-7712758eee23)

![image](https://github.com/user-attachments/assets/77346d47-9b56-44b7-b6d4-28c39a56f500)

![image](https://github.com/user-attachments/assets/7432e8a5-eb98-4a21-ae47-f986c5d2b8cc)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the campaign builder and add conditions + decisions to see they're normal
3. Open Assets > New > Select Remote storage > Click to Browse files > Close the modal > Open again
4. Open Account and click to Reset learning experience, a flash message will appear to show it's working
5. Open Contacts > actions dropdown > export to CSV > see flash notification with X for closing
